docs/development/pigs/pig-21.rst

```diff
@@ -0,0 +1,348 @@
+.. include:: ../../references.txt
+
+.. _pig-021:
+
+**************************************
+PIG 21 - Models Framework Improvements
+**************************************
+
+* Author: Axel Donath, Régis Terrier and Quentin Remy
+* Created: Jun 10, 2020
+* Accepted:
+* Status:
+* Discussion:
+
+Abstract
+========
+This PIG outlines further minor improvements to the modeling framework of Gammapy.
+This includes introduction of a spectra models with norm parameters, minimal
+support for energy dependent spatial models as well as simplifications of the
+YAML serialisation.
+
+
+Proposal
+========
+
+Spectral Norm Models
+--------------------
+
+For the purpose of adjusting template based models we propose to introduce a new
+class of spectral models. Those spectral models feature a norm parameter instead
+of amplitude and are named using the ``NormSpectralModel`` suffix:
```

Comment:
Thanks @registerrier for the comment here. Here are a few remarks from my side:

- I fully agree with the spatial multiplicative models, however there is technically no need to introduce a special class for it (except for a `CompoundSpatialModel` or similar), because we can just re-use our exiting spatial models. They don't have an amplitude parameter and always return a "norm" already. In a way this is an inconsistency, because they don't have a "norm" suffix in the name.
- I think the core of the problem here is that the "amplitude" parameter is bound to the `SpectralModel` class. Which prevents a "clean use" as a model to scale other existing models (which was the only motivation to introduce the `NormSpectralModel`...). Maybe we re-evaluate the solution to attach the "amplitude" to the `SkyModel`?

Reply:
Possibly we can also check whether we can simplify the way to re-use our normal `SpectralModel`. As I have shown before, this works:

```
amplitude = Parameter("amplitude", unit="")
pwl = PowerLawSpectralModel(amplitude=amplitude, index=2)
```
Do you think it's an improvement if we could get this to work:
```
pwl = PowerLawSpectralModel(amplitude=1, index=0)
```
?